src/views/pages/Login/index.js

```diff
@@ -18,11 +19,13 @@ class LoginPage extends Component {
         this.state = {
             username: '',
             password: '',
-            logout: null
+            logout: null,
+            loginFailed: false
         };
     }
 
     componentDidMount() {
+        this.setState({ loginFailed: false });
```

Comment:
Is there a reason we're initialing state in didMount? This will cause a re-render and I'm pretty sure it should work without this.

Reply:
Not a particular reason why it's there - I just wanted to reset loginFailed, so the message wouldn't persist. I was noticing while testing that it seemed to persist. Maybe it could be set in 'componentWillMount'?

Reply:
It’s unnecessary for both because you’re setting it false in the constructor which is called at relatively similar places in the lifecycle. It’s an anti pattern to call setState in either willMount or didMount and will mount especially as it’s deperecated in the version of React we are using

Reply:
It must've just been weird because of it being developed, and the state seeming to persist when the app would recompile.